cockpit/templates/api/api-pdb.yaml

```diff
@@ -0,0 +1,25 @@
+{{- if and (.Values.api.enabled) (.Values.api.pdb.enabled) -}}
+apiVersion: {{ include "common.capabilities.policy.apiVersion" . }}
+kind: PodDisruptionBudget
+metadata:
+  name: {{ template "gravitee.api.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ template "gravitee.name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/version: {{ .Values.api.image.tag | default .Chart.AppVersion | quote }}
+    app.kubernetes.io/component: "{{ .Values.api.name }}"
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+spec:
+  {{- if .Values.api.pdb.maxUnavailable }}
+  maxUnavailable: {{ .Values.api.pdb.maxUnavailable }}
+  {{ end -}}
+  {{- if .Values.api.pdb.minAvailable }}
+  minAvailable: {{ .Values.api.pdb.minAvailable }}
+  {{ end -}}
```

Comment:
Shouldn't we drive the content with the Values?
Other we would need to upgrade the chart to support more settings 🤔 

```
 {{ toYaml .Values.api.pdb | nindent 4 -}}
```

Reply:
I would say why not, but PDB is very straightforward, not a lot you can do https://docs.openshift.com/container-platform/4.9/rest_api/policy_apis/poddisruptionbudget-policy-v1.html